core/condition.go

```diff
@@ -22,6 +22,11 @@ const (
 	// Values that could replace the placeholder: 127.0.0.1, 10.0.0.1, ...
 	IPPlaceholder = "[IP]"
 
+	// DNSRCodePlaceHolder is a place holder for DNS_RCODE
+	//
+	// Values that could be NOERROR, FORMERR, SERVFAIL, NXDOMAIN, NOTIMP and REFUSED
+	DNSRCodePlaceHolder = "[DNS_RCODE]"
```

Comment:
`DNSRCodePlaceHolder` should be `DNSRCodePlaceholder`

This one's a bit my fault, I noticed `Placeholder` and `PlaceHolder` were used inconsistently in the code so I just replaced all `PlaceHolder` by `Placeholder`

Reply:
it's mine, even you mentioned `Placeholder`, I used `PlaceHolder `. 😅